Podverse/Podcast.swift

```diff
@@ -36,9 +34,18 @@ class Podcast: NSManagedObject {
         self.mutableSetValue(forKey: "episodes").remove(value)
     }
     
-    static func retrieveMediaRefsFromServer(episodeMediaUrl: String? = nil, podcastFeedUrl: String? = nil, onlySubscribed: Bool? = nil, completion: @escaping (_ mediaRefs:[MediaRef]?) -> Void) {
+    static func retrievePodcastByFeedUrl(feedUrlString: String) -> Podcast? {
```

Comment:
retrievePodcastByFeedUrl right now is intended to handle the situation when we're on the FindSearchTableVC and we unsubscribe from a podcast. The podcasts in the FindSearchTableVC are search results of the PodcastSearchResult class, not of the Podcast managed object class.

If on FindSearchTableVC we unsubscribe from a podcast (PodcastSearchResult), how do we tell CoreData to delete that corresponding Podcast locally? retrievePodcastByFeedUrl is meant to handle that situation (used in PVDeleter line 31).

---

Also, more generally, a substantial change from the previous Podverse is I'm not longer adding "isSubscribed" flags to local Podcast objects. If you're subscribed then the podcast is saved in CoreData. If you're not subscribed, the podcast is deleted from CoreData.